### PR TITLE
Support HWASan in .options file with [hwasan] section.

### PR DESCRIPTION
**Pull request by inferno-chromium**
Fixes https://github.com/google/clusterfuzz/issues/1611

### CODE REVIEW

**Comment by inferno-chromium** [comment]
Leon from skia team was asking on this.

File: src/python/bot/fuzzers/options.py [hunk 1]
```diff
@@ -156,6 +156,10 @@ def get_ubsan_options(self):
     """Return a list of UBSAN_OPTIONS overrides."""
     return self._get_option_section('ubsan')
 
+  def get_hwasan_options(self):
+    """Return a list of UBSAN_OPTIONS overrides."""
```
Comment (Dor1s) [inline src/python/bot/fuzzers/options.py:160]: nit: typo in `UBSAN_OPTIONS`

**Comment by inferno-chromium** [comment]
> The change LGTM, thanks! But I'm surprised that we want to re-use `[asan]` section for HWASan. I think the original request for this came in from Mitch, who had something in mind for Android.
> 
> Why don't we want to use an explicit `[hwasan]` section?

yes, thinking again, it feels much cleaner to use hwasan section rather than re-use asan. i was thinking cross-platform stuff, but i think build pipelines should be different per-platform. So, lets keep it cleaner. so fixed it now.

File: src/python/bot/fuzzers/options.py (hunk 1 above)
Comment (inferno-chromium) [reply src/python/bot/fuzzers/options.py]: Done!

**Comment by Dor1s** [comment]
and PR title can be changed now (since we don't rely on `[asan]` anymore). Thanks a lot!

**Pull request merged**